extensions/cornerstone/src/commandsModule.ts

```diff
@@ -145,6 +146,45 @@ function commandsModule({
   }
 
   const actions = {
+    loadSegmentationsForActiveViewport: () => {
+      const { displaySetService, userAuthenticationService } = servicesManager.services;
+
+      const getDerivedSequences = (displaySetUID: string): DisplaySet[] => {
+        const currentDs = displaySetService.getDisplaySetByUID(displaySetUID);
+        const displaySetCache = displaySetService.getDisplaySetCache();
+        return Array.from(displaySetCache.values()).filter(
+          (ds: any): ds is DisplaySet =>
+            (ds?.referencedDisplaySetInstanceUID === displaySetUID ||
+              ds.referencedSeriesInstanceUID === currentDs.SeriesInstanceUID) &&
+            !ds?.isHydrated
+        );
+      };
+
```

Comment:
I don't see any logic to check if this is derived. I think derived DS have isDerived on them or similar